Whoaverse/Whoaverse/Views/Shared/_MarkdownEditor.cshtml

```diff
@@ -40,68 +40,68 @@
+        @*<a class="markdownEditorTextButton" alt="emoji" title="Sleeping Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0],'😀','');">😴</a>*@
+        <a class="markdownEditorTextButton" alt="emoji" title="Dizzy Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😵', '');">😵</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Face Without Mouth" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😶', '');">😶</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Face With Medical Mask" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😷', '');">😷</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Grinning Cat Face With Smiling Eyes" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😸', '');">😸</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Cat Face With Tears Of Joy" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😹', '');">😹</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Smiling Cat Face With Open Mouth" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😺', '');">😺</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Smiling Cat Face With Heart-shaped Eyes" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😻', '');">😻</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Cat Face With Wry Smile" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😼', '');">😼</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Kissing Cat Face With Closed Eyes" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😽', '');">😽</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Pouting Cat Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😾', '');">😾</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Crying Cat Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😿', '');">😿</a>
+        <a class="markdownEditorTextButton" alt="emoji" title="Weary Cat Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '🙀', '');">🙀</a>
+        @*<a class="markdownEditorTextButton" alt="emoji" title="Slightly Frowning Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0],'🙁','');">🙁</a>
```

Comment:
Encoding issues, notice the squares for the characters on the commented out lines. I was going to have DF look and fix what he can then delete the remaining.

-----Original Message-----
From: "Atko" <notifications@github.com>
Sent: ‎1/‎6/‎2015 4:36 PM
To: "whoaverse/whoaverse" <whoaverse@noreply.github.com>
Cc: "PuttItOut" <github.com@d.786k.com>
Subject: Re: [whoaverse] Lost Encoding Replacements (#229)

In Whoaverse/Whoaverse/Views/Shared/_MarkdownEditor.cshtml:
> +        <a class="markdownEditorTextButton" alt="emoji" title="Astonished Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😲', '');">😲</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Flushed Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😳', '');">😳</a>
> +        @*<a class="markdownEditorTextButton" alt="emoji" title="Sleeping Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0],'😀','');">😴</a>*@
> +        <a class="markdownEditorTextButton" alt="emoji" title="Dizzy Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😵', '');">😵</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Face Without Mouth" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😶', '');">😶</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Face With Medical Mask" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😷', '');">😷</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Grinning Cat Face With Smiling Eyes" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😸', '');">😸</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Cat Face With Tears Of Joy" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😹', '');">😹</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Smiling Cat Face With Open Mouth" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😺', '');">😺</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Smiling Cat Face With Heart-shaped Eyes" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😻', '');">😻</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Cat Face With Wry Smile" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😼', '');">😼</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Kissing Cat Face With Closed Eyes" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😽', '');">😽</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Pouting Cat Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😾', '');">😾</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Crying Cat Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '😿', '');">😿</a>
> +        <a class="markdownEditorTextButton" alt="emoji" title="Weary Cat Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0], '🙀', '');">🙀</a>
> +        @*<a class="markdownEditorTextButton" alt="emoji" title="Slightly Frowning Face" onclick="addTagsToSelectedText($(this.parentElement.parentElement.parentElement).find('textarea')[0],'🙁','');">🙁</a>
Are you still working on this? May I ask what's up with these commented-out lines? Can we drop them?
—
Reply to this email directly or view it on GitHub.=